virtcontainers/network.go

```diff
@@ -75,6 +80,9 @@ func (n *NetInterworkingModel) SetModel(modelName string) error {
 	case "enlightened":
 		*n = NetXConnectEnlightenedModel
 		return nil
+	case "tcfilter":
```

Comment:
I know it's not only related to this PR, but we should define a constant for this. We don't want to play around with strings like this.